classes/test.php

```diff
@@ -1826,6 +1830,14 @@ private static function cleanNamespace($namespace)
 
 	private static function isRegex($namespace)
 	{
-		return preg_match('/^([^\\\[:alnum:][:space:]]).*\1.*$/', $namespace) === 1;
+		return is_numeric(@preg_match($namespace, 'foo'));
```

Comment:
Instead of `is_numeric` I would have tested the return value of `preg_match` against `false` which is what `preg_match` returns in case of error.